feature-layer-update-attributes/src/main/java/com/esri/arcgisruntime/sample/featurelayerupdateattributes/MainActivity.java

```diff
@@ -56,9 +66,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // create a map with the streets basemap
     final Map map = new Map(Basemap.createStreets());
-    //set an initial viewpointf
+
+    //set an initial viewpoint
     map.setInitialViewpoint(new Viewpoint(new Point(544871.19, 6806138.66, SpatialReferences
         .getWebMercator()), 2E6));
```

Comment:
Is there a reason you set the initial viewpoint instead of simplifying by using the `Map(Basemap.Type, lat, lon, lod)`?